cmd/siac/hostdbcmd.go

```diff
@@ -37,11 +38,13 @@ var (
 	}
 
 	hostdbSetFiltermodeCmd = &cobra.Command{
-		Use:   "setfiltermode [filtermode] [host] [host] [host]...",
+		Use:   "setfiltermode [filtermode] [host,...] [netaddress,...]",
 		Short: "Set the filtermode.",
-		Long: `Set the hostdb filtermode and specify hosts.
+		Long: `Set the hostdb filtermode and specify comma separated hosts and net addresses.
```

Comment:
perhaps instead, we could leave it as `[host] [host] [host]`, but allow `host` to be *either* a pubkey or a net address. That way, the existing behavior is preserved. It also avoids the ugliness of *only* specifying net addresses (which would otherwise require passing `""` for the list of pubkeys).